lib/net.js

```diff
@@ -146,6 +146,8 @@ function Socket(options) {
   } else if (options.fd !== undefined) {
     this._handle = createHandle(options.fd);
     this._handle.open(options.fd);
+    // Non-strict equations here for legacy reasons.
```

Comment:
Instead of saying this and linking to a GitHub issue, could you just explain the legacy reasons.